pyproject.toml

```diff
@@ -36,6 +36,7 @@ dynamic = [
 dependencies = [
   "astropy>=5.0.1",
   "h5py>=3.1",
+  "ipykernel>=6.6.1",
```

Comment:
This is needed for the plasma calculator, so it should be a required dependency.